modules/phase_field/src/kernels/CoupledSusceptibilityTimeDerivativeInterface.C

```diff
@@ -0,0 +1,73 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "CoupledSusceptibilityTimeDerivativeInterface.h"
+
+registerMooseObject("PhaseFieldApp", CoupledSusceptibilityTimeDerivativeInterface);
+
+template <>
+InputParameters
+validParams<CoupledSusceptibilityTimeDerivativeInterface>()
+{
+  InputParameters params = validParams<CoupledSusceptibilityTimeDerivative>();
+  params.addClassDescription(
+      "A modified coupled time derivative Kernel that multiplies the time "
+      "derivative of a coupled variable by a generalized susceptibility and interface normal");
```

Comment:
we will not normally be multiplying by a susceptibility, instead of susceptibility it will be a material property that is the product of the the composition differencebetween phases and the appropriate prefactors